apps/aecore/src/aec_peers_pool.erl

```diff
@@ -1141,11 +1147,12 @@ verified_add(St, Now, Peer) ->
             Peer2 = get_peer(St3, PeerId),
             Peer3 = Peer2#peer{vidx = BucketIdx},
             St4 = set_peer(St3, PeerId, Peer3),
-            case is_available(St4, PeerId) of
+            St5 = add_lookup_verif_all(St4, PeerId),
+            case is_available(St5, PeerId) of
```

Comment:
`is_available` can also return `undefined` - the callers `(un)verified_add` should handle that result value?

Reply:
yes, it's handled in `verified_maybe_add`.